File: C#/C# Basics/First Steps in Coding/Exercise Tasks/Task3_DepositCalculator.cs
using System;

namespace Task3_DepositCalculator
{
    class Task3_DepositCalculator
    {
        static void Main(string[] args)
        {
            double deposit = double.Parse(Console.ReadLine());
            int months = int.Parse(Console.ReadLine());
            double annualPercent = double.Parse(Console.ReadLine());


            double interest = deposit * annualPercent / 100;
            double monthlyInterest = interest / 12;
            double sum = deposit + (months * monthlyInterest);

            Console.WriteLine(sum);
         
        }
    }
}

File: C++/Basics/First Steps in Coding/Exercise Tasks/Task3_DepositCalculator.cpp
// Task3_DepositCalculator.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>


using namespace std;

int main()
{
    double deposit;
    cin >> deposit;

    int months;
    cin >> months;

    double percent;
    cin >> percent;


    double interest = percent * deposit;
    double monthlyInterest = interest / 12;

    double sum = deposit + (3 * ((deposit * percent) / 100) / 12); 

    cout.setf(ios::fixed);
    cout.precision(2);
    cout << sum; 
    return 0;


  
}
